### ISSUE TITLE
About Res50 accuracy on ShanghaiTechB

### ISSUE THREAD

**Issue by ckcraig01** [gjy3035/C-3-Framework#51]
Dear Author, thank you fro providing this awesome framework. I have trained the Res50 model with ShangHaiTechB dataset but only get MAE with 8.4 instead of 7.7 (reported), is there any hyperparameter setting that required?

**Comment by haoranD** [comment]
Thanks for your amazing working @gjy3035. I also got a result which around MAE 8.4. Could you please give us any suggestion?

Thanks for your question. @ckcraig01  I got a quite similar result to you. Could you please give me any suggestion?

Much appreciated.

**Comment by gjy3035** [comment]
maybe compare the setting.py and config.py in the results reports folder.

**Comment by ckcraig01** [comment]
Thank you author, and @haoranD  thanks. I have reviewed the setting.py/config.py, found that the differences are only on the size of val batch and the training epoch. 
but from "12-28_09-18_SHHB_Res50_1e-05.txt" I found from the log, the training can reach 7.7MAE in 200 epochs. So @haoranD please let me know if you have other discover

**Comment by gjy3035** [comment]
I upload the code that achieves MAE of 7.7, you can run it. Note that it is a python2 version. 

[code.zip](https://github.com/gjy3035/C-3-Framework/files/3958910/code.zip)

**Comment by haoranD** [comment]
Cool, working on it. Cheers
